app/models/manageiq/providers/kubernetes/container_manager/event_catcher_mixin.rb

```diff
@@ -55,17 +55,23 @@ def queue_event(event)
   end
 
   def filtered?(event)
-    extract_event_data(event).nil?
+    parsed_event = extract_event_data(event)
+    parsed_event.nil? || filtered_events.include?(parsed_event[:event_type])
```

Comment:
@Fryguy I found [this](https://github.com/ManageIQ/manageiq/blob/1a4bcfb05f9cc37ac76d310bf857a63f7352d0fc/app/models/manageiq/providers/google/cloud_manager/event_catcher/runner.rb#L42-L45) that could leverage that as well (it seems very similar code).
I have the feeling that this filtering logic should be in the core `EventCatcher` and we may have just to provide something like `parse_event_type`.